src/config/dynamic-theme-fixes.config

```diff
@@ -10472,6 +10472,7 @@ mesa3d.org
 
 INVERT
 .collapsed a svg
+a svg
```

Comment:
Could this be _specified_ a bit further? E.g. adding a specific class-name? `a svg` is too _generic_ and could unwanted target other elements.

Reply:
Would it be the most specific if I put the following:

================================

mesa3d.org

INVERT
[data-bs-toggle="collapse"].collapsed > a > svg
[data-bs-toggle="collapse"] > a > svg

================================

This way both lines in the fix for mesa3d.org are corrected to be as specific as possible?

Reply:
That would indeed be more specific! But `[data-bs-toggle="collapse"].collapsed > a > svg` would be redundant then. so you can replace it with a one-liner :D